### PR TITLE
add new hook to add new template in comment

### PR DESCRIPTION
**Pull request by seiya-annie**
### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

What's Changed: add a new hook to give template in the comment when user input /info

How it Works: in the comment input box, input '/info'

### Release note 
No release note

### CODE REVIEW

File: pkg/providers/add-template/event.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package add_template
+
+import (
+	"github.com/google/go-github/v32/github"
+	"github.com/pingcap-incubator/cherry-bot/util"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"regexp"
+	"strings"
```
Comment (zhouqiang-cl) [inline pkg/providers/add-template/event.go:9]: group this import?

File: pkg/providers/add-template/event.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	if err := c.processComment(event, event.Comment.GetBody()); err != nil {
+		util.Error(err)
+	}
+}
+
+func (c *Comment) processComment(event *github.IssueCommentEvent, comment string) error {
+	issueID := event.GetIssue().GetNumber()
+	temMatches := templatePattern.FindStringSubmatch(comment)
+	if strings.TrimSpace(temMatches[0]) == "/info" {
+		return errors.Wrap(c.addTemplate(issueID), "label issue")
```
Comment (zhouqiang-cl) [inline pkg/providers/add-template/event.go:29]: Here will always return error?

File: pkg/providers/add-template/event.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	"io/ioutil"
+	"regexp"
+	"strings"
+)
+
+var (
+	templatePattern = regexp.MustCompile(`\/info`)
+)
+
+func (c *Comment) ProcessIssueCommentEvent(event *github.IssueCommentEvent) {
```
Comment (zhouqiang-cl) [inline pkg/providers/add-template/event.go:16]: When this function will be called

Comment (you06) [reply pkg/providers/add-template/event.go:16]: `bot/webhook.go` may need to be updated.

**Comment by you06** [comment]
IMO the `template.txt` may be specified from config file and not in code repo.

File: pkg/providers/add-template/event.go (hunk 2 above)
Comment (seiya-annie) [reply pkg/providers/add-template/event.go]: sorry, my mistake, will fix it

File: pkg/providers/add-template/event.go (hunk 3 above)
Comment (seiya-annie) [reply pkg/providers/add-template/event.go:16]: thanks, I will add this process in bot main function

**Comment by seiya-annie** [comment]
> IMO the `template.txt` may be specified from config file and not in code repo.

OK, I will try to fix this

**Review by zhouqiang-cl — COMMENTED** [review]
LGTM
@you06 @AndreMouche PTAL

**Comment by seiya-annie** [comment]
> IMO the `template.txt` may be specified from config file and not in code repo.

我看了一下，我直接把这个文件从repo里删除，然后放到和生成的 bot 文件同一个目录就行吧。我直接删除吧。

File: pkg/providers/add-template/event.go (hunk 1 above)
Comment (seiya-annie) [reply pkg/providers/add-template/event.go:7]: fixed it

**Comment by you06** [comment]
Mostly LGTM, can you remove some unnecessary comments from code?

**Comment by you06** [comment]
Thanks for this PR!

**Pull request merged**